config.py

```diff
@@ -0,0 +1,12 @@
+
+# Configuation file
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+# Attributes of the database:
+CSRF_ENABLED = True
```

Comment:
1.We will delete this line then.
2.the current secret key is temporary. We will add a script not available on github and put the info of the key inside it in a later stage.
3.We have config.cfg file but not included in this pull request. We will commit it soon.